doc/user-guide.md

```diff
@@ -569,9 +572,9 @@ spec:
     certificate: {}
 ```
 
-In this scenario the operator will generate `Certificate` resource with common name of `myapp.test.svc` that can be used for service to service communication.
+In this scenario the operator will generate `Certificate` resource with common name of `my-app.test.svc` that can be used for service to service communication.
 
-Once this certificate request is resolved by cert-manager the resulting secret `myapp-svc-tls` will be 
+Once this certificate request is resolved by cert-manager the resulting secret `my-app-svc-tls` will be 
```

Comment:
Change to "After this certificate request is resolved by the certificate manager, the resulting `my-app-svc-tls` secret  is mounted onto each pod inside the `/etc/x509/certs` folder. Mounted files are always up to date with a secret.".